.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "*.astro": ["prettier --write"]
```

Comment:
Issue currently is that prettier --check has it signal the file but not the line where the improper formatting is.